ecole-doctorale-de-sciences-po-author-date-french.csl

```diff
@@ -0,0 +1,440 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="fr-FR" version="1.0" page-range-format="expanded" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Ecole doctorale de Sciences Po (author-date, French)</title>
+    <id>http://www.zotero.org/styles/ecole-doctorale-de-sciences-po-author-date-french</id>
+    <link href="http://www.zotero.org/styles/ecole-doctorale-de-sciences-po-author-date-french" rel="self"/>
+    <author>
+      <name>Anna Couthures-Idrizi</name>
+      <email>anna.couthuresidrizi@sciencespo.fr</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <category field="generic-base"/>
+    <updated>2019-01-10T00:59:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="ordinal-01">ère</term>
+      <term name="ordinal-02">e</term>
+      <term name="ordinal-03">e</term>
+      <term name="ordinal-04">e</term>
+      <term name="cited">Op.&#160;cit.</term>
+      <term name="page" form="short">p.</term>
+	<term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+<macro name="contributors-note">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="short" and="text" delimiter-precedes-last="never" sort-separator=" " font-style="normal">
+          </name>
+        </names>
+      </if>
+      <else-if variable="editor">
+        <names variable="editor">
+          <name form="short" and="text" delimiter-precedes-last="never" sort-separator=" " font-style="normal">
+          </name>
+          <label form="short" prefix="&#160;(" suffix=".)"/>
+        </names>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <choose>
+      <if variable="author">
+        <names variable="author">
+          <name form="long" and="text" name-as-sort-order="first" sort-separator=", " font-style="normal">
+            <name-part name="family" font-variant="small-caps"/>
+          </name>
```

Comment:
ditto here -- editor should just be in substitute

Reply:
i don't understand how to do that

Reply:
could you help me, please ?

Reply:
Dear all, for a lot of items, i just copied/pasted elements from other styles already in your repository. I do not understand why i should have to correct those elements. 
If it's really necessary to correct them, could you please help me ?
Best regards,
Anna

Reply:
Yes, if you want a style hosted by us, which goes out to software with hundreds of thousands of end users, we absolutely insist on quality control on submission. If you want to have the style available, you can always self-host it. 

We do this on a volunteer basis, so if you need significant help -- as in this case, where you essentially ask me to explain almost every change request -- this may take a bit longer, sorry.

Reply:
Dear Sebastian,
Thank you _so much_ for your help !
I'm already volunteer for other academic subjects, and i'm well aware that people write you only when they have a problem or a claim ; so i deeply apologize if my last email hurted you, that was not the scope.
Now that you fixed all issues, what is the next step ? Can both styles (author date & note) be shared through styles repository ?
Best regards,
Anna